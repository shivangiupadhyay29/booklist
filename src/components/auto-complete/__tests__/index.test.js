import React from 'react';
import AutoComplete from '../index.js';
import { shallow } from 'enzyme';

describe('Autocomplete component', () => {
  it('autocomplete component unit testing', () => {
    const mockFnHandler = jest.fn();
    const limit = 2;
    const wrapper = shallow(
      <AutoComplete limit={limit} addSelectedBook={mockFnHandler} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
