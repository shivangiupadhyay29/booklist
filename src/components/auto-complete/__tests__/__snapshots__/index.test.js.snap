// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Autocomplete component autocomplete component unit testing 1`] = `ShallowWrapper {}`;
